tests/data/generate_data.ipynb

```diff
@@ -0,0 +1,314 @@
+    "- IDs are integers from 700-831."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hipscat_import.pipeline as runner\n",
+    "from hipscat_import.catalog.arguments import ImportArguments\n",
+    "from hipscat_import.index.arguments import IndexArguments\n",
+    "from hipscat_import.margin_cache.margin_cache_arguments import MarginCacheArguments\n",
+    "import tempfile\n",
+    "from pathlib import Path\n",
+    "\n",
+    "tmp_path = tempfile.TemporaryDirectory()\n",
+    "tmp_dir = tmp_path.name\n",
+    "\n",
+    "hipscat_import_dir = \"../../../hipscat-import/tests/hipscat_import/data/\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### small_sky\n",
+    "\n",
+    "This \"object catalog\" is 131 randomly generated radec values. \n",
+    "\n",
+    "- All radec positions are in the Healpix pixel order 0, pixel 11.\n",
+    "- IDs are integers from 700-831.\n",
+    "\n",
+    "This catalog was generated with the following snippet:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "args = ImportArguments(\n",
+    "    input_path=Path(hipscat_import_dir) / \"small_sky\",\n",
+    "    output_path=\".\",\n",
+    "    file_reader=\"csv\",\n",
+    "    output_artifact_name=\"small_sky\",\n",
+    "    overwrite=True,\n",
+    "    tmp_dir=tmp_dir,\n",
+    ")\n",
+    "runner.pipeline(args)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### small_sky_order1\n",
+    "\n",
+    "This catalog has the same data points as other small sky catalogs,\n",
+    "but is coerced to spreading these data points over partitions at order 1, instead\n",
+    "of order 0.\n",
+    "\n",
+    "This means there are 4 leaf partition files, instead of just 1, and so can\n",
+    "be useful for confirming reads/writes over multiple leaf partition files.\n",
+    "\n",
+    "NB: Setting `constant_healpix_order` coerces the import pipeline to create\n",
+    "leaf partitions at order 1.\n",
+    "\n",
+    "This catalog was generated with the following snippet:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "args = ImportArguments(\n",
+    "    input_path=Path(hipscat_import_dir) / \"small_sky\",\n",
+    "    output_path=\".\",\n",
+    "    file_reader=\"csv\",\n",
+    "    output_artifact_name=\"small_sky_order1\",\n",
+    "    constant_healpix_order=1,\n",
+    "    overwrite=True,\n",
+    "    tmp_dir=tmp_dir,\n",
+    ")\n",
+    "runner.pipeline(args)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### small_sky_order1_id_index\n",
+    "\n",
+    "An index table (that is NOT a \"true\" hipscat catalog) to map the \"id\" field to the partition the row can be found in."
```

Comment:
Not sure that it matters, but you don't say "This catalog was generated using the following snippet:" in this block.